wallet/cosmos/client.go

```diff
@@ -0,0 +1,154 @@
+package cosmos
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	authv1beta1 "cosmossdk.io/api/cosmos/auth/v1beta1"
+	"github.com/armon/go-metrics"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"google.golang.org/grpc"
+)
+
+type Client struct {
+	grpcConn        *grpc.ClientConn
+	bankClient      banktypes.QueryClient
+	txServiceClient tx.ServiceClient
+	authClient      authv1beta1.QueryClient
+	bankKeeper      keeper.BaseKeeper
+}
+
+func NewClient(rpcTarget string) (*Client, error) {
```

Comment:
use config for rpcTarget， like other chain, and should use client array for this NewClient

Reply:
done. use array outside the client, let the adaptor make the choice to select rpc server